### ISSUE TITLE
 Importing pytables (3.6.1) in python 3.7.8 fails for me in windows system

### ISSUE THREAD

**Issue by Sophia409** [PyTables/PyTables#824]
Hi,

After installing and uninstalling pytables packages for many times, I still couldn't import tables. 

Hope for your help

```
(base) C:\Users\yuhong>conda list | grep pytables
pytables                  3.6.1            py37h14417ae_3    conda-forge  
(base) C:\Users\yuhong>python

Python 3.7.8 | packaged by conda-forge | (default, Jul 31 2020, 01:53:57) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tables
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\yuhong\AppData\Roaming\Python\Python37\site-packages\tables\__init__.py", line 99, in <module>
    from .utilsextension import (
ImportError: DLL load failed: The specified procedure could not be found.
```

**Comment by exrich** [comment]
+1.  I have the same issue on python 3.7.9.  This seems to be related to blosc and perhaps a module called snappy.  Downgrading to blosc=1.20.0 caused snappy to be reinstalled and fixed the issue for me.

**Comment by avalentino** [comment]
@Sophia409, @exrich thanks for reporting.
As far as I can understand the issue seems to be mainly related to conda packaging of pytables.
Probably it is a good idea to open a corresponding issue on https://github.com/conda-forge/pytables-feedstock.

**Comment by e-dism** [comment]
I have the same problem by trying to import tables in a python 3.7.9 environment. It seems, that the DLLs are not delivered correctly via pip. Installing the matching wheel file from https://www.lfd.uci.edu/~gohlke/pythonlibs/#pytables let me import tables without error messages.

**Comment by avalentino** [comment]
@Sophia409 the issue seems to be very similar to #913.
Please try to re-install in a fresh conda environment.

@e-dism the issue with wheels has been solved in #926.
